deploy/cloudwash_job.yaml

```diff
@@ -57,6 +58,16 @@ objects:
                       secretKeyRef:
                         key: subscription_id
                         name: provisioning-azure-auth-secret
+                  - name: CLEANUP_AZURE__AUTH__RESOURCE_GROUPS
+                    valueFrom:
+                      secretKeyRef:
+                        key: resource_group
+                        name: cloudwash-secret
```

Comment:
copypasta from your local env - the secre is called `provisioning-azure-auth-secret`